controllers/disruption_controller.go

```diff
@@ -38,7 +38,7 @@ import (
 	"go.opentelemetry.io/otel/trace"
 	"go.uber.org/zap"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
This change came about while testing, happy to undo it, but I think it helps make it clear in the code that we aren't importing the std lib's `errors` package